src/plugins/plugins_ex.cpp

```diff
@@ -121,3 +121,25 @@ std::string FieldToString(const std::map<uint64_t, std::string>& maps, uint64_t
 
 // Errors
 char ERROR_MSG_ADDING_TRAP[] = "Failed to add a trap";
+
+pluginex::pluginex(drakvuf_t drakvuf, output_format_t output)
+    : m_output_format(output), drakvuf(drakvuf)
+{};
+
+pluginex::~pluginex()
+{
+    stop();
```

Comment:
CodeQL warning: Virtual call from constructor or destructor

Reply:
This code was not written by me - it's already preset on master branch. I just moved implementations to .cpp file to make it more readable.